### ISSUE TITLE
Regression: Argument -c no longer works for column index

### ISSUE THREAD

**Issue by awendt** [benmaier/binpacking#24]
Hi! 👋🏻 

I think I found a bug.

## Steps to reproduce

1.  Create a CSV file:
    ```csv
    module,duration
    a,17
    b,13
    c,13
    c,14
    ```
2. Run `binpacking -f /path/to.csv -c duration -H -N 3`
3. Run `binpacking -f /path/to.csv -c 1 -H -N 3`
4. Observe

## Expected behavior

Both commands output:

```
=== distributed items to bins with sizes ===
0 17.0
1 14.0
2 26.0
```

## Actual behavior

Using `-c 1` raises:

```
Traceback (most recent call last):
  File "/usr/local/bin/binpacking", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/binpacking/binpacking_binary.py", line 68, in main
    csv_to_constant_bin_number(**opt)
  File "/usr/local/lib/python3.9/site-packages/binpacking/to_constant_bin_number.py", line 29, in csv_to_constant_bin_number
    data, weight_column, header = load_csv(filepath,
  File "/usr/local/lib/python3.9/site-packages/binpacking/utilities.py", line 30, in load_csv
    raise Exception("weight key "+weight_column+" not found in header")
TypeError: can only concatenate str (not "int") to str
```

This is a regression from 1.4.3 which showed expected behavior.

**Comment by benmaier** [comment]
thanks! fixing this now

**Comment by benmaier** [comment]
fixed in #25. This seems to have been broken quite some time ago. Thanks, @awendt !
